src/app/personside/infotabs/meldinger/MeldingerContainer.tsx

```diff
@@ -34,6 +39,18 @@ const MeldingerArticleStyle = styled.article`
     position: relative;
 `;
 
+function TraadVisningWrapper(props: Props) {
```

Comment:
Aha, du bruker de her ja. Når du kaller et  interface for Props er det implisitt at dette er Propsene til den eksporterte komponenten. Her har du laget en liten "hjelpekomponent", og da bør du nok kalle interfacet noe noe mindre generisk, feks TraadVisningWrapperProps